cmds/core/pidof/pidof.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2025 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build linux
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"strings"
+)
+
+var errNotFound = errors.New("pid name not found")
+
+type proc struct {
+	comm string
```

Comment:
If I had to guess, does `comm` mean `commandline`? If this is the case, IMHO `cmdline` would be a more descriptive naming.

Reply:
not really, `cmdline` is different thing and it as well exist under ${PID} directory, so it better to stick with `comm`.

Reply:
You are right, I wrote this comment prematurely, in the procfs ot is named comm. I still think you should put a short command on that according to [man 5 proc_pid_com](https://man7.org/linux/man-pages/man5/proc_pid_comm.5.html) that it is the `command name`. In my experience `comm` is also used in context of communication which is here clearly not the case. But since this is just an internal structure it might not be too important...